File: General/34 - Change Feature State/FeatureStateChange v3.0/cboItem.vb
Imports System.Collections.Generic
Imports System.Text

    Class cboItem
        Private _value As String

        Private _name As String
        Public Property Value() As String

            Get
                Return _value
            End Get
            Set(ByVal value As String)
                _value = value
            End Set
        End Property

        Public Property Name() As String


            Get
                Return _name
            End Get
            Set(ByVal value As String)
                _name = value
            End Set
        End Property

        Public Sub New(ByVal name As String, ByVal value As String)


            _name = name


            _value = value
        End Sub
        Public Overrides Function ToString() As String

            Return _name

        End Function
    End Class

File: Fiber/125 - Update OLTID for FDC/Update OLTID for FDC v1.4.1/Source Code/Update FDC v1.4.1/cboItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NEPS.GTechnology.UpdateFDC
{
    class cboItem
    {
        private string _value;

        private string _name;
        public string Value

        {
        get { return _value; }set { _value = value; }

        }
        public string Name

        {

        get { return _name; }
        set { _name = value; }

        }
        public cboItem(string name, string value)

        {

        _name = name;

        _value = value;

        }
        public override string ToString()

        {
        return _name;

        }
    }
}
